pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -228,8 +255,69 @@ func (r *ReconcileIBMBlockCSI) Reconcile(request reconcile.Request) (reconcile.R
 	return reconcile.Result{}, nil
 }
 
+func (r *ReconcileIBMBlockCSI) addFinalizerIfNotPresent(instance *ibmblockcsi.IBMBlockCSI) error {
+	logger := log.WithName("addFinalizer")
```

Comment:
Is there any convention for the logger text?
In this case, it's only the function's prefix.

Reply:
I don't think we have an official convention, though in my current change it's usually the full name of the function.
do you think it would be better to include its full name here?

Reply:
done

Reply:
I think so. Otherwise, it may confuse the log readers.